python/examples/quickstarters/calendar_agent/crewai/main.py

```diff
@@ -33,8 +45,22 @@
 
 
 # Create and Execute Agent.
-def run_crew():
-    calendar_agent = Agent(
+def run_crew() -> Tuple[str, int]:
```

Comment:
Great improvement in type safety and documentation:
1. Added return type annotation `-> Tuple[str, int]`
2. Added comprehensive docstring with Returns and Raises sections
3. Added proper type hints for variables

This makes the code more maintainable and helps catch type-related bugs early.